### ISSUE TITLE
Applet Bug

### ISSUE THREAD

**Issue by radiantcactus** [linuxmint/cinnamon-spices-applets#6204]
### Applet version/Build date

1.7.6

### Cinnamon version

6.0.4

### Distribution

Mint 21.3

### Graphics hardware *and* driver used

NVIDIA RTX A2000 Mobile driver version 535.183.01

### Applet name and maintainer

qredshift@quintao @raphaelquintao

### What happened?

When I adjust temp settings while Enabled, the screen goes black and I can only see the cursor. The only way out is to manually power off my machine.

### Other information

System is a Thinkpad P17 Gen 2, i7-11800H 32GB ram, not sure if that info helps. Thanks!

**Comment by raphaelquintao** [comment]
Have you tried to update to the new version? Current version now is 2.0.4. 

You use the following command to do that: 
```shell
curl https://github.com/raphaelquintao/QRedshiftCinnamon/raw/master/install.sh -sSfL | bash 
```

Let me know if it works for you.

**Comment by radiantcactus** [comment]
Yes, it was the new version with the problem. I actually rolled back to the
version on Mint Applets menu and it worked.
